src/handlers/shared/start.ts

```diff
@@ -109,5 +123,5 @@ export async function start(context: Context, issue: Context["payload"]["issue"]
     ].join("\n") as string
   );
 
-  return { output: "Task assigned successfully" };
+  return { content: "Task assigned successfully", status: "ok" };
```

Comment:
```suggestion
  return { content: "Task assigned successfully", statusCode: 200 };
```

HTTP status codes are far more concise and precise. Consider using these?

Reply:
It is arbitrary and not related to HTTP requests, this is just info for the Kernel, but sure can change it.

Reply:
It makes sense to follow that standard.